src/hotspot/share/prims/jvm.cpp

```diff
@@ -3913,9 +3916,11 @@ JVM_ENTRY(void, JVM_VirtualThreadMountEnd(JNIEnv* env, jobject vthread, jboolean
   if (JvmtiExport::should_post_vthread_mount()) {
     JvmtiExport::post_vthread_mount(vthread);
   }
+#endif
```

Comment:
It would be bug if these functions were called when JVMTI isn't compiled in so maybe we should put an #else ShouldNotReachHere() to catch it.

Reply:
I think that makes sense. At first I was thinking why not just #ifdef out the whole API, but these APIs are mapped natives for `VirtualThread.notifyJvmtiMountBegin()` and `VirtualThread.notifyJvmtiMountEnd()`. Calls to them come from java, but should only happen if jvmti is enabled.

Reply:
I agree, it is a good idea. Of course, it makes sense to add for all 4 native functions:
 JVM_VirtualThreadMountBegin, JVM_VirtualThreadMountEnd,
 JVM_VirtualThreadUnmountBegin, JVM_VirtualThreadUnmountEnd,

Reply:
All right, see new commit.